Project.toml

```
@@ -16,7 +16,7 @@ ReTestItems = "1"
 Sockets = "1"
 Test = "1"
 julia = "1.8"
-object_store_ffi_jll = "0.6.0"
+object_store_ffi_jll = ">=0.6.5"
```

why `>=`? this probably not what we want, since it's declaring compatibility with all possible future versions. do we not just want `0.6.5` up to `0.7.0`?
```suggestion
object_store_ffi_jll = "0.6.5"
```

---

Fixed! Thanks